.github/CODEOWNERS

```diff
@@ -0,0 +1,3 @@
+# This is the squad name responsible for this project.
+# It will be automatically assigned to opened PR.
+* @canonical/Desktop
```

Comment:
I'm unsure what team(s) to put here. Since this repository is under Ubuntu, it would probably have to be an ubuntu team as well for write access reasons.

Alternatively, we don't use codeowners at all and go on a individual-basis.